components/table/table-col-sort-button.js

```diff
@@ -14,6 +14,7 @@ import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
 /**
  * Button for sorting a table column in ascending/descending order.
  * @slot - Text of the sort button
+ * @slot items - Multi-facted sort items. Generally assigned to the slot attribute on a nested d2l-table-col-sort-button-item.
```

Comment:
Not an accessibility thing, but is there a `click` event? Or I guess we're just piggy-backing off the internal `button`'s click event... but maybe we should be documenting it anyway?

Reply:
Yup it would be piggy-backing off the internal `button` click event. I can add documentation for that.